### DISCUSSION TITLE
Welcome and our roadmap here.

### DISCUSSION THREAD

**Discussion by hlhr202** [Atome-FE/llama-node#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## Roadmap
We are going to put our roadmap here. Some of them can be solved in near future, but some of them need further investigation and pending our dependency update.

- [ ] CLI tools (WIP)
- [ ] Support first onnx model! (WIP)
- [ ] Parallel inference
- [ ] Bloom support (by llm-rs)
- [ ] More platforms and architectures
- [ ] Embedding and semantic search (by bert.cpp or onnx bert)
- [ ] GPU inference

**Comment by linonetwo** [comment]
will there be GPU support using project like https://github.com/harrisonvanderbyl/rwkv-cpp-cuda ?

**Reply by hlhr202** [reply]
hi, honestly i dont have time for this😂. but will keep an eye for more gpu inference projects. currently i m more focusing on the ggml related project and probably will try wonnx in the next step. but anyway if you are interest on this, i also welcome PR. thanks.
